src/main/java/org/jfrog/hudson/pipeline/scripted/steps/CollectEnvStep.java

```diff
@@ -37,7 +40,9 @@ public Execution(CollectEnvStep step, StepContext context) throws IOException, I
 
         @Override
         protected Boolean run() throws Exception {
```

Comment:
I see no point in returning Boolean. I'd prefer to change to Void and return null.
```suggestion
        protected Void run() throws Exception {
```

Reply:
`run()` is being overridden, therefore we should keep the same signature of the origin func.

Reply:
Yes, but it can be fixed easily by changing 
```java
public static class Execution extends ArtifactorySynchronousNonBlockingStepExecution<Boolean> 
```
to
```java
public static class Execution extends ArtifactorySynchronousNonBlockingStepExecution<Void> 
```